addons/binding/org.openhab.binding.nest/src/main/java/org/openhab/binding/nest/internal/NestUtils.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.nest.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+import com.google.gson.FieldNamingPolicy;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+/**
+ * Utility class for sharing utility methods between objects.
+ *
+ * @author Wouter Born - Initial contribution
+ */
+@NonNullByDefault
+public class NestUtils {
+
+    public static final Gson GSON = new GsonBuilder().setDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'")
```

Comment:
I believe @kaikreuzer and OSGI do not like statics :-)

Reply:
I know that's the case with loggers. What's the policy with utility classes like this one? The binding constants also have a lot of static variables. Why is that OK?

Reply:
a `GSON` instance is of course different from a static `String` which its own place anyway. But for other objects types in the binding constants it is fair question.

Reply:
As input for this discussion I searched for other static variables with a Gson instance and found that ESH also has some, namely:
* org.eclipse.smarthome.core.events.AbstractEventFactory.JSONCONVERTER
* org.eclipse.smarthome.config.core.status.events.ConfigStatusInfoEvent.GSON

So if it is considered a bad practice, these may also need some modification. WDYT @kaikreuzer ?

Reply:
Static fields in a util class are acceptable. The difference here to the places in ESH that you mention is that you declare that field publicly and that can cause issues as other bundles might keep a reference to it (and that can cause lifecycle issues if bundles are updated, uninstalled, etc.).

If you'd create a static method `NestUtils.fromJson(...)` and keep the GSON instance private, this would imho be the better solution.